src/cmd/services/m3query/config/config.go

```diff
@@ -228,6 +231,46 @@ func (c QueryConfiguration) TimeoutOrDefault() time.Duration {
 	return defaultQueryTimeout
 }
 
+// RestrictTagsAsStorageRestrictByTag returns restrict tags as
+// storage options to restrict all queries by default.
+func (c QueryConfiguration) RestrictTagsAsStorageRestrictByTag() (*storage.RestrictByTag, bool, error) {
```

Comment:
Is this bool return value ever used? It seems to be discarded in all callers.

Reply:
It is not, however I really dislike sometimes just returning `nil` and an error. It forces the caller to think about the fact that sometimes it can be `nil` when error is not returned but the bool is `false`.